src/scales/scale.category.js

```diff
@@ -92,10 +90,28 @@ export default class CategoryScale extends Scale {
 		return me.getPixelForValue(index * me._numLabels / ticks.length + me.min);
 	}
 
+	/**
+	 * @param {number} pixel
+	 * @return {number}
+	 */
 	getValueForPixel(pixel) {
-		const me = this;
-		const value = Math.round(me._startValue + me.getDecimalForPixel(pixel) * me._valueRange);
-		return Math.min(Math.max(value, 0), me.ticks.length - 1);
+		return Math.round(this.getValueForDecimal(this.getDecimalForPixel(pixel)));
```

Comment:
I think this is slightly different than before. The previous code clamped the output to be >= 0 and < ticks.length.

Reply:
Ticks can be auto skipped, so I think thats wrong.

Reply:
![image](https://user-images.githubusercontent.com/27971921/87238694-82b97580-c40e-11ea-9e08-973aecb181f2.png)